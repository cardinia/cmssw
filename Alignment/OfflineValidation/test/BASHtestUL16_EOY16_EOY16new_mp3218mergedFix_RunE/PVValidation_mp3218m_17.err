27-Sep-2019 21:44:57 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/20063FC9-AC07-E911-8717-24BE05C6E711.root
27-Sep-2019 21:45:01 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/20063FC9-AC07-E911-8717-24BE05C6E711.root
Begin processing the 1st record. Run 277070, Event 268456274, LumiSection 147 on stream 0 at 27-Sep-2019 21:45:14.838 CEST
Begin processing the 1001st record. Run 277070, Event 275920042, LumiSection 151 on stream 0 at 27-Sep-2019 21:48:46.050 CEST
Begin processing the 2001st record. Run 277070, Event 282679298, LumiSection 155 on stream 0 at 27-Sep-2019 21:51:51.740 CEST
Begin processing the 3001st record. Run 277070, Event 296492760, LumiSection 162 on stream 0 at 27-Sep-2019 21:55:11.201 CEST
Begin processing the 4001st record. Run 277070, Event 302719090, LumiSection 166 on stream 0 at 27-Sep-2019 21:58:27.017 CEST
Begin processing the 5001st record. Run 277070, Event 310543645, LumiSection 170 on stream 0 at 27-Sep-2019 22:01:31.388 CEST
Begin processing the 6001st record. Run 277070, Event 319890591, LumiSection 175 on stream 0 at 27-Sep-2019 22:04:42.268 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:05:27 CEST  Run: 277070 Event: 321626163
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 7001st record. Run 277070, Event 326275966, LumiSection 179 on stream 0 at 27-Sep-2019 22:07:40.689 CEST
Begin processing the 8001st record. Run 277070, Event 334488455, LumiSection 183 on stream 0 at 27-Sep-2019 22:10:18.123 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:10:45 CEST  Run: 277070 Event: 335995764
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 578
%MSG
Begin processing the 9001st record. Run 277070, Event 342998381, LumiSection 188 on stream 0 at 27-Sep-2019 22:12:53.665 CEST
Begin processing the 10001st record. Run 277070, Event 351475719, LumiSection 192 on stream 0 at 27-Sep-2019 22:15:30.755 CEST
Begin processing the 11001st record. Run 277070, Event 360000542, LumiSection 197 on stream 0 at 27-Sep-2019 22:18:11.449 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:19:59 CEST  Run: 277070 Event: 365960300
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 37
%MSG
Begin processing the 12001st record. Run 277070, Event 366596201, LumiSection 201 on stream 0 at 27-Sep-2019 22:20:50.276 CEST
Begin processing the 13001st record. Run 277070, Event 377249719, LumiSection 206 on stream 0 at 27-Sep-2019 22:23:34.219 CEST
Begin processing the 14001st record. Run 277070, Event 384475222, LumiSection 210 on stream 0 at 27-Sep-2019 22:26:17.681 CEST
Begin processing the 15001st record. Run 277070, Event 392935830, LumiSection 215 on stream 0 at 27-Sep-2019 22:28:54.824 CEST
Begin processing the 16001st record. Run 277070, Event 400350060, LumiSection 219 on stream 0 at 27-Sep-2019 22:31:37.059 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:32:14 CEST  Run: 277070 Event: 402271459
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 2500
%MSG
Begin processing the 17001st record. Run 277070, Event 408273268, LumiSection 224 on stream 0 at 27-Sep-2019 22:34:22.181 CEST
Begin processing the 18001st record. Run 277070, Event 416968817, LumiSection 228 on stream 0 at 27-Sep-2019 22:37:07.085 CEST
Begin processing the 19001st record. Run 277070, Event 425038015, LumiSection 233 on stream 0 at 27-Sep-2019 22:40:21.647 CEST
Begin processing the 20001st record. Run 277070, Event 434337338, LumiSection 238 on stream 0 at 27-Sep-2019 22:43:33.542 CEST
Begin processing the 21001st record. Run 277070, Event 442039091, LumiSection 242 on stream 0 at 27-Sep-2019 22:46:33.903 CEST
Begin processing the 22001st record. Run 277070, Event 450715269, LumiSection 247 on stream 0 at 27-Sep-2019 22:49:32.813 CEST
Begin processing the 23001st record. Run 277070, Event 458246148, LumiSection 252 on stream 0 at 27-Sep-2019 22:52:25.196 CEST
Begin processing the 24001st record. Run 277070, Event 465392840, LumiSection 256 on stream 0 at 27-Sep-2019 22:55:22.724 CEST
Begin processing the 25001st record. Run 277070, Event 474017371, LumiSection 260 on stream 0 at 27-Sep-2019 22:58:26.406 CEST
Begin processing the 26001st record. Run 277070, Event 482257650, LumiSection 265 on stream 0 at 27-Sep-2019 23:01:38.815 CEST
Begin processing the 27001st record. Run 277070, Event 490947876, LumiSection 270 on stream 0 at 27-Sep-2019 23:04:55.439 CEST
Begin processing the 28001st record. Run 277070, Event 499060693, LumiSection 274 on stream 0 at 27-Sep-2019 23:08:14.968 CEST
Begin processing the 29001st record. Run 277070, Event 506926019, LumiSection 279 on stream 0 at 27-Sep-2019 23:11:07.346 CEST
Begin processing the 30001st record. Run 277070, Event 514425593, LumiSection 283 on stream 0 at 27-Sep-2019 23:13:58.816 CEST
Begin processing the 31001st record. Run 277070, Event 523960067, LumiSection 288 on stream 0 at 27-Sep-2019 23:16:51.891 CEST
Begin processing the 32001st record. Run 277070, Event 529475403, LumiSection 292 on stream 0 at 27-Sep-2019 23:19:48.120 CEST
Begin processing the 33001st record. Run 277070, Event 538885338, LumiSection 297 on stream 0 at 27-Sep-2019 23:22:51.104 CEST
27-Sep-2019 23:23:42 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/20063FC9-AC07-E911-8717-24BE05C6E711.root
27-Sep-2019 23:23:42 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/022E97C8-F007-E911-8600-003048F316A4.root
27-Sep-2019 23:23:45 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/022E97C8-F007-E911-8600-003048F316A4.root
Begin processing the 34001st record. Run 277070, Event 816900138, LumiSection 438 on stream 0 at 27-Sep-2019 23:25:58.350 CEST
Begin processing the 35001st record. Run 277070, Event 826355010, LumiSection 443 on stream 0 at 27-Sep-2019 23:28:49.056 CEST
Begin processing the 36001st record. Run 277070, Event 834486474, LumiSection 447 on stream 0 at 27-Sep-2019 23:31:42.276 CEST
Begin processing the 37001st record. Run 277070, Event 842136429, LumiSection 451 on stream 0 at 27-Sep-2019 23:34:47.240 CEST
Begin processing the 38001st record. Run 277070, Event 850447199, LumiSection 455 on stream 0 at 27-Sep-2019 23:37:44.263 CEST
Begin processing the 39001st record. Run 277070, Event 858648446, LumiSection 459 on stream 0 at 27-Sep-2019 23:40:44.718 CEST
Begin processing the 40001st record. Run 277070, Event 866670563, LumiSection 463 on stream 0 at 27-Sep-2019 23:44:00.449 CEST
Begin processing the 41001st record. Run 277070, Event 876819798, LumiSection 468 on stream 0 at 27-Sep-2019 23:47:00.463 CEST
Begin processing the 42001st record. Run 277070, Event 883874651, LumiSection 472 on stream 0 at 27-Sep-2019 23:49:52.363 CEST
Begin processing the 43001st record. Run 277070, Event 892490048, LumiSection 476 on stream 0 at 27-Sep-2019 23:52:47.258 CEST
Begin processing the 44001st record. Run 277070, Event 898782457, LumiSection 480 on stream 0 at 27-Sep-2019 23:55:56.044 CEST
Begin processing the 45001st record. Run 277070, Event 906443239, LumiSection 484 on stream 0 at 27-Sep-2019 23:58:59.835 CEST
Begin processing the 46001st record. Run 277070, Event 914628595, LumiSection 488 on stream 0 at 28-Sep-2019 00:01:57.843 CEST
Begin processing the 47001st record. Run 277070, Event 923473013, LumiSection 492 on stream 0 at 28-Sep-2019 00:04:52.831 CEST
Begin processing the 48001st record. Run 277070, Event 929584303, LumiSection 496 on stream 0 at 28-Sep-2019 00:08:11.476 CEST
Begin processing the 49001st record. Run 277070, Event 939598806, LumiSection 501 on stream 0 at 28-Sep-2019 00:11:25.586 CEST
28-Sep-2019 00:14:31 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/022E97C8-F007-E911-8600-003048F316A4.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       4        4
    2 fileAction           -s file_close                             2        2
    3 fileAction           -s file_open                              4        4

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          277070/321626163 277070/335995764 277070/402271459
    2 fileAction           PostProcessEvent PostGlobalEndRun 
    3 fileAction           pre-events       pre-events       PostProcessEvent

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 4                   4
System                  6                   6

dropped waiting message count 0
[9.955MB/9.955MB][100%][==================================================][9.955MB/s]  [9.955MB/9.955MB][100%][==================================================][9.955MB/s]  
