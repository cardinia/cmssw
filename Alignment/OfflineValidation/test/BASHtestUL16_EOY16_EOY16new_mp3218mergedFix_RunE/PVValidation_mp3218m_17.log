000 (3196552.017.000) 09/27 21:32:53 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196552.017.000) 09/27 21:44:38 Started transferring input files
	Transferring to host: <188.184.197.176:9618?addrs=188.184.197.176-9618+[2001-1458-301-103--100-81]-9618&noUDP&sock=15799_931c_65495>
...
040 (3196552.017.000) 09/27 21:44:38 Finished transferring input files
...
001 (3196552.017.000) 09/27 21:44:38 Job executing on host: <188.184.197.176:9618?addrs=188.184.197.176-9618+[2001-1458-301-103--100-81]-9618&noUDP&sock=5712_890e_93>
...
006 (3196552.017.000) 09/27 21:44:47 Image size of job updated: 50724
	33  -  MemoryUsage of job (MB)
	32800  -  ResidentSetSize of job (KB)
...
006 (3196552.017.000) 09/27 21:49:48 Image size of job updated: 670788
	541  -  MemoryUsage of job (MB)
	553752  -  ResidentSetSize of job (KB)
...
006 (3196552.017.000) 09/27 21:54:49 Image size of job updated: 670788
	541  -  MemoryUsage of job (MB)
	553760  -  ResidentSetSize of job (KB)
...
006 (3196552.017.000) 09/27 21:59:49 Image size of job updated: 671676
	542  -  MemoryUsage of job (MB)
	554640  -  ResidentSetSize of job (KB)
...
006 (3196552.017.000) 09/27 22:19:51 Image size of job updated: 674872
	545  -  MemoryUsage of job (MB)
	557800  -  ResidentSetSize of job (KB)
...
006 (3196552.017.000) 09/27 23:24:58 Image size of job updated: 679560
	550  -  MemoryUsage of job (MB)
	562448  -  ResidentSetSize of job (KB)
...
006 (3196552.017.000) 09/27 23:29:59 Image size of job updated: 680548
	551  -  MemoryUsage of job (MB)
	563436  -  ResidentSetSize of job (KB)
...
006 (3196552.017.000) 09/27 23:55:01 Image size of job updated: 681824
	552  -  MemoryUsage of job (MB)
	564712  -  ResidentSetSize of job (KB)
...
006 (3196552.017.000) 09/28 00:00:02 Image size of job updated: 682372
	553  -  MemoryUsage of job (MB)
	565260  -  ResidentSetSize of job (KB)
...
040 (3196552.017.000) 09/28 00:14:54 Started transferring output files
...
040 (3196552.017.000) 09/28 00:14:54 Finished transferring output files
...
005 (3196552.017.000) 09/28 00:14:54 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 02:27:20, Sys 0 00:00:11  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 02:27:20, Sys 0 00:00:11  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	10646  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	10646  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :    10243        1   2852880 
	   Memory (MB)          :      553     2000      2000 
...
