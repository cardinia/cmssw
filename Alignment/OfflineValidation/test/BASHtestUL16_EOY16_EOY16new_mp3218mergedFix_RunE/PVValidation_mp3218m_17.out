Job started at  Fri Sep 27 21:44:38 CEST 2019
batch dir: /pool/condor/dir_11842 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_277070_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 11M -rw-r--r--. 1 tkello zh 10M Sep 28 00:14 PVValidation_mp3218m_277070.root -rw-r--r--. 1 tkello zh 21K Sep 27 21:44 PVValidation_mp3218m_277070_cfg.py -rw-r--r--. 1 tkello zh 9.2K Sep 28 00:14 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 27 21:44 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 27 21:44 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:44 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:44 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:44 var
Job ended at  Sat Sep 28 00:14:54 CEST 2019
